docs/install/managing-multiple-projects.md

```diff
@@ -1,3 +1,103 @@
+
+In order to make your PubSubSource in Project A' cluster to consume Google PubSub resource in Project B:
+1. Control Plane GSA `events-controller-gsa@project-a.iam.gserviceaccount.com` needs `pubsub.editor` role for `project-b`
+    ```
+    gcloud projects add-iam-policy-binding project-b
+    --member=serviceAccount:events-controller-gsa@project-a.iam.gserviceaccount.com
+    --role roles/pubsub.editor
+    ```
+    ***Note:*** To create different Google Cloud Resource, the Control Plane needs different permissions.
+    Refer to [Manually Configure Authentication Mechanism for the Control Plane](./authentication-mechanisms-gcp.md/#authentication-mechanism-for-the-control-plane)
+    for specific roles(a set of permissions) needed to create each Source.
+
+1. Source Data-plane GSA `your-gsa@project-b.iam.gserviceaccount.com` needs `pubsub.editor` role for `project-b`
+
+    ```
+    gcloud projects add-iam-policy-binding project-b
+    --member=serviceAccount:your-gsa@project-b.iam.gserviceaccount.com
+    --role roles/pubsub.editor
+    ```
+   ***Note:*** Source Data Plane only pulls subscription from Cloud PubSub Subscription, thus,
+   no matter which Cloud Source you are creating, this GSA only needs `roles/pubsub.editor` role.
+   Refer to [Installing a Service Account for the Data Plane](../install/dataplane-service-account.md)
+   for specific roles(a set of permissions) needed if you also want metrics and tracing for your Source.
+
+1. Configure credentials in the kubernetes cluster. You can either use:
+    1. Workload Identity
+        1. Create a Kubernetes Service Account `ksa-name` in the same namespace where the PubSubSource will reside.
+            ```
+            kubectl create service account ksa-name -n namespace
+            ```
+        2. Source Data-plane GSA `your-gsa@project-b.iam.gserviceaccount.com` needs workload identity binding.
+            ```
+            gcloud iam service-accounts add-iam-policy-binding your-gsa@project-b.iam.gserviceaccount.com
+            --role "roles/iam.workloadIdentityUser"
+            --member "serviceAccount:project-a.svc.id.goog[namespace/ksa-name]”
+            ```
+        3. Kubernetes Service Account `ksa-name` needs annotation
+            ```
+            kubectl annotate serviceaccount ksa-name \
+            iam.gke.io/gcp-service-account=your-gsa@project-b.iam.gserviceaccount.com
+            ```
+    1. or Exporting Service Account Keys And Store Them as Kubernetes Secret
+       1. Download a new JSON private key for Source Data-plane GSA `your-gsa@project-b.iam.gserviceaccount.com`
+
+          ```shell
+          gcloud iam service-accounts keys create events-sources-key.json \
+            --iam-account=your-gsa@project-b.iam.gserviceaccount.com
+          ```
+
+       1. Create a secret on the Kubernetes cluster with the downloaded key. Remember
+          to create the secret in the namespace your resources will reside.
```

Comment:
Not sure what "resource" means here. We can use `project-a` or `project-b` instead.